package-lock.json

```diff
@@ -0,0 +1,1690 @@
+{
```

Comment:
i think package-lock should be in .gitignore and not checked in

Reply:
What if the dependencies have been upgraded by other package owner and this change may cause our project incompatible?

Reply:
Okay, I see the .gitignore file of autorest.az repo and autorest.cli repo